packages/tocco-ui/src/utilStyles/declareInteractionColors.js

```diff
@@ -1,21 +1,21 @@
 import {theme} from 'styled-system'
 
-const declareInteractionColors = colors => {
+const declareInteractionColors = (colors, format = 'html') => {
```

Comment:
why dont you create a const with this
   ${format === 'html' ? 'background-color' : 'fill'}:
and use it instead of using the ternary operator on every line. the code should get better readable.


also ..can you think of something better than this magic string "html"? Maybe move to a enum like object oder just rename the parameter to isHtml?

Reply:
Possible formats are "html" and "svg". I moved format definition to stylingConstants and introduced tenary operator fillProperty and strokeProperty.